tests/run-pass/file_read.rs

```diff
@@ -2,10 +2,16 @@
 // compile-flags: -Zmiri-disable-isolation
 
 use std::fs::File;
-use std::io::Read;
+use std::io::{ Read, Write };
 
 fn main() {
-    // FIXME: create the file and delete it when `rm` is implemented.
+    // FIXME: remove the file and delete it when `rm` is implemented.
+
+    // Test creating, writing and closing a file (closing is tested when `file` is dropped).
```

Comment:
Check that opening a nonexisting file fails as expected

Reply:
Hmmm aparently we need another intrinsic for this 
```
error[E0080]: Miri evaluation error: can't call foreign function: __errno_location
  --> /home/christian/.rustup/toolchains/miri/lib/rustlib/src/rust/src/libstd/sys/unix/os.rs:63:11
   |
63 |         (*errno_location()) as i32
   |           ^^^^^^^^^^^^^^^^ Miri evaluation error: can't call foreign function: __errno_location
   |
```

Reply:
Oh heh, OK, not in this PR then